cmd/oauth2_implicit.go

```diff
@@ -25,10 +25,13 @@ func (c *OAuth2Cmd) ImplicitGrantFlow(clientConfig oauth2.ClientConfig, serverCo
 
 	LogRequest(authorizeRequest)
 
-	Logfln("\nOpen the following URL:\n\n%s\n", authorizeRequest.URL.String())
+	Logfln("\nGo the following URL:\n\n%s\n", authorizeRequest.URL.String())
 
-	if err = browser.OpenURL(authorizeRequest.URL.String()); err != nil {
-		LogError(err)
+	if !clientConfig.NoBrowser {
+		Logfln("Opening browser...\n")
```

Comment:
I would rather drop this log

Reply:
I find `Opening browser...` actually useful, otherwise it's not clear what `oauth2c` is doing in case of a problem:
```
...
Opening browser...

  ERROR   exec: "xdg-open,x-www-browser,www-browser,wslview": executable file not found in $PATH
...
```
Without this line, the user would only see this error without a clue as to what the cause is.